rem/aufile/aufile.c

```diff
@@ -278,3 +278,18 @@ size_t aufile_get_length(struct aufile *af, struct aufile_prm *prm)
 
 	return 0;
 }
+
+/**
+ * Set initial playing position of a WAV file in ms
+ *
+ * @param af  Audio-file
+ * @param prm Audio file parameters from aufile_open
+ *
+ * @return position in bytes or (size_t)-1 in case of an error.
+ */
+size_t aufile_set_position(struct aufile *af, struct aufile_prm *prm, size_t pos_ms)
+{
+	size_t pos = prm->srate * aufmt_sample_size(prm->fmt) * prm->channels * pos_ms / 1000;
```

Comment:
This is only an approximation, right? Is it a good approximation for all WAV files?

Shouldn't the WAV header size be added?

Reply:
Thanks & yes. The wav header needs to be taken into account.